### DISCUSSION TITLE
daily agenda email

### DISCUSSION THREAD

**Discussion by eleaner** [Kozea/Radicale#1557] [General]
hi,

Does anyone know how to do / what to use to receive daily email with the planned events?
I am replacing the google calendar and that's about the only thing I am missing.

thanks

**Comment by pbiering** [comment]
_radicale_ is currently not designed for sending or receiving e-mails, see also https://github.com/Kozea/Radicale/discussions/1394

Potenially a cron job can help, eventually using the new implemented free/busy report feature by https://github.com/Kozea/Radicale/pull/1337 -> @ray-react0r : can you comment and if so, can you provide an example e.g. for the Wiki?

**Comment by ray-react0r** [comment]
The free/busy report only reports times with which you are free or busy. It won't include event descriptions for privacy reasons so I don't think the free/busy report will help here. It's mostly used for sharing a report so that people can work out schedules. 

Personally, I'd keep this out of Radicale and write an external app to query the server and send out emails.

**Reply by eleaner** [reply]
I am up to use an external app, 
but not really to write one
any suggestions?

**Reply by ray-react0r** [reply]
Unfortunately, I don't. A quick search didn't provide me with anything useful. I only found libraries that would require coding.

**Reply by pbiering** [reply]
a _groupware_ should be able to support this and other e-mail requirements - _radicale_ is only a simple and lightweight Cal/CardDAV server (so far).

**Comment by eleaner** [comment]
I ended up extending someones's Python script
intended to run from cron and seems to be working

https://github.com/eleaner/caldav-daily-digest
